src/opm/output/eclipse/Summary.cpp

```diff
@@ -1356,6 +1359,21 @@ void Summary::add_timestep( int report_step,
                             const std::map<std::string, std::vector<double>>& region_values,
                             const std::map<std::pair<std::string, int>, double>& block_values) {
 
+    if (secs_elapsed < this->prev_time_elapsed) {
```

Comment:
I guess this should be `<=`?

Reply:
Maybe.  It's **probably** okay to have `secs_elapsed == this->prev_secs_elapsed`&mdash;at least that does not (greatly) affect the cumulative quantities, but I can add a strict requirement that
```C++
secs_elapsed > this->prev_secs_elapsed
```
if you'd like.

Reply:
IIRC, you'll get NaNs due to a division by zero in either the storage or the flux term in the simulator. Also, how many zero-length time step does it require to finish simulating a day? ;)

Reply:
> Also, how many zero-length time step does it require to finish simulating a day

I expect at most one zero-length time step, but I'm on the fence on whether or not to support it.  I'm thinking of the case in which the calling code (i.e., the simulator) for whatever reason calls `add_timestep()` using the restarted solution itself before advancing the process.  If we don't want to support that case then enforcing a strictly increasing sequence of `secs_elapsed` is the right choice.